pkg/cmd/cli/install/install.go

```diff
@@ -130,6 +132,54 @@ func NewCommand(f client.Factory) *cobra.Command {
 	return c
 }
 
+func CheckVersion(image string) bool {
```

Comment:
The version check will return false if the version is v2.0.0. That's not right. So, please refer to Internet for some idea about version check. I believe there is a bunch utilities available.

BTW, the next version of CSI controller is just v2.0.0